src/Server.js

```diff
@@ -234,24 +235,24 @@ app.get("/course/:id", (request, response) => {
     id: courseId
   };
   handler.sendMessage("riak", JSON.stringify(riakData));
-
-  const uri = encodeURI(`${MONGO_HOST}/course/${courseId}`);
+  const uri = encodeURI(`${REDIS_HOST}/course/${courseId}`);
   const options = {
-    method: "GET",
-    uri: uri,
-    json: true
-  };
+    method = "GET",
+    uri = uri,
+    json = true
```

Comment:
won't work. json object is `{ field: value }`

Reply:
good point.